src/types/dsnpTypes.ts

```diff
@@ -0,0 +1,103 @@
+import { EthereumAddress } from "@liberty30/lib-privacy-js";
+
+// Broadcast: a public post
+// fromAddress	ID of the sender	bytes20
+// messageID	keccak-256 hash of content stored at URI	bytes32
+// uri	content URI	string
+export interface DSNPBroadcast {
+  fromAddress: EthereumAddress;
+  messageID: Uint8Array;
+  uri: string;
+}
```

Comment:
We need these comments to match docblock style so they generate the docs nicely:

```suggestion
/**
 * Broadcast: a public post
 */
export interface DSNPBroadcast {
  /** ID of the sender */
  fromAddress: EthereumAddress;
  /** keccak-256 hash of content stored at URI */
  messageID: Uint8Array;
  /** content URI */
  uri: string;
}
```